tests/admin_views/tests.py

```diff
@@ -1305,6 +1305,16 @@ def test_initial_data_can_be_overridden(self):
         # this is the overridden behaviour
         self.assertContains(response, 'value="overridden_value"')
 
+    def test_formset_params_can_be_overridden(self):
+        """
+        The behavior for setting initial formset parameters can be overridden in the
+        ModelAdmin class.
+        """
+        response = self.client.get(
+            reverse('admin:admin_views_region_add', current_app=self.current_app),
+        )
+        self.assertContains(response, 'default_country_value')
```

Comment:
The docstring is superfluous.

```suggestion
    def test_formset_params_can_be_overridden(self):
        response = self.client.get(
            reverse('admin:admin_views_state_add', current_app=self.current_app),
        )
        self.assertContains(response, 'default_country_value')
```

(Although you probably want to update `'default_country_value'` now that we're changing models.)